sda/internal/config/config.go

```diff
@@ -356,22 +335,12 @@ func NewConfig(app string) (*Config, error) {
 		if err != nil {
 			return nil, err
 		}
-	case "backup":
-		c.configArchive()
-		c.configBackup()
-
-		err := c.configBroker()
-		if err != nil {
-			return nil, err
-		}
-
-		err = c.configDatabase()
-		if err != nil {
-			return nil, err
+	case "finalize":
```

Comment:
i don't see a backup service any more maybe we should remove it with this PR if it is removed in config

Reply:
The functionality of the backup service is needed in BigPicture together with the corresponding API component in order to replicate datasets between the two sites, although under a different name so people can't mix it up.
Once everything related FEGA is done I'll create the BP speciffic PRs.

Reply:
I see, ok.